### PR TITLE
adding new field in the mental health form section

### PR DESCRIPTION
**Pull request by Dutervil**
@lnball This branch contains 6 tickets. they are all ready to be reviewed

### CODE REVIEW

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 1]
```diff
@@ -793,7 +793,76 @@
...
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.PoorIntrospection"/>
+                    </label>
+                    <obs conceptId="CIEL:166051" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:814]: @Dutervil I recommend that you reword the question OR answers.  

The question/label should be "Introspection" not "Poor introspection".  I think the answers should be "normal" and abnormal.

If you want the question to be "Poor introspection" with y/n, you can use that same concept and label with concepts of answers of "Poor" (yes label) and "normal" (no label)

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 2]
```diff
@@ -793,7 +793,76 @@
...
+                </p>
+
+            </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.ThoughtContent"/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:821]: Your label is "Thought content" but the obs/question is "Muscle tone and strength"

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 3]
```diff
@@ -793,7 +793,76 @@
...
+
+            </div>
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.suicidal"/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:834]: The label and obs don't match.  Label is Suicidal but the obs/question is "Thought content".  yes/no are not the right answers.

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 4]
```diff
@@ -793,7 +793,76 @@
...
+                    <obs conceptId="PIH:14157" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.phobia"/>
+                    </label>
+                    <obs conceptId="CIEL:121543" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:845]: Perfect.

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 5]
```diff
@@ -793,7 +793,76 @@
...
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.Psychosocial"/>
+                    </label>
+                    <obs conceptId="PIH:13175" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:858]: Perfect

**Review by lnball — CHANGES_REQUESTED** [review]
3 of the questions are wrong.  I commented on github AND JIRA.

File: configuration/pih/htmlforms/mentalHealth.xml (hunk 3 above)
Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:834]: What should be the type, @lnball ?

Comment (lnball) [reply configuration/pih/htmlforms/mentalHealth.xml:834]: @Dutervil I don't think that "thought content" has a yes/no answer.  It's usually asked with abnormal thoughts as answers (ie. delusions, negativism, paranoid delusion, denies suicidal thoughts, denies homicidal thoughts.).  See this example for Liberia.  
image
If your users want a simple radio, I would recommend answers of normal/abnormal.

**Review by lnball — CHANGES_REQUESTED** [review]
@Dutervil You fixed everything except "Thought content".  See my comments.

**Comment by Dutervil** [comment]
Hi @lnball how are you doing. So I use CIEL:167112 for Thought content. it gives me a drop-down , different  from the one from Liberia , as shown above ` <obs conceptId="CIEL:167112" style="checkbox"  answerSeparator=""/>`
Screenshot from 2024-01-08 08-38-50

File: configuration/pih/htmlforms/mentalHealth.xml [hunk 6]
```diff
@@ -820,7 +820,7 @@
                     <label>
                         <uimessage code="pihcore.mentalhealth.ThoughtContent"/>
                     </label>
-                    <obs conceptId="CIEL:167112" style="radio" answerConceptIds="PIH:NORMAL,PIH:ABNORMAL"  answerSeparator=""/>
+                    <obs conceptId="CIEL:167112" style="checkbox"  answerSeparator=""/>
```
Comment (lnball) [inline configuration/pih/htmlforms/mentalHealth.xml:823]: If you want checkboxes, you would add 1 obs (line) for each checkbox and include the answerConceptId.  For example, `<obs conceptId="CIEL:167112" style="checkbox" answerConceptId="CIEL:142600" /> <!-- Delusions checkbox -->`

Comment (Dutervil) [reply configuration/pih/htmlforms/mentalHealth.xml:823]: I do not understand @lnball . I text you on google Meet so we can discuss

**Review by lnball — APPROVED** [review]
@Dutervil That's good.  Hopefully Matthieu can help with adding the right question/answers.

**Pull request merged**